store/tikv/backoff.go

```diff
@@ -233,6 +234,9 @@ func (b *Backoffer) Backoff(typ backoffType, err error) error {
 			}
 		}
 		log.Warn(errMsg)
+		if strings.Contains(errMsg, ErrMismatchClusterID.Error()) {
```

Comment:
Can we define a string constant for "mismatch cluster id"?

Reply:
Is it better to check and handle this in the pd-client? @disksing What do you think about it?

Reply:
I think we can check it at the beginning of this function.

Reply:
Good idea.